contracts/SpokePool.sol

```diff
@@ -1035,6 +1094,10 @@ abstract contract SpokePool is
             relayExecution.maxTokensToSend,
             (relayData.realizedLpFeePct + relayExecution.updatedRelayerFeePct)
         );
+        // If fill amount minus fees, which is possible with small fill amounts and negative fees, then
+        // revert.
+        require(fillAmountPreFees > 0, "fill amount pre fees is 0");
```

Comment:
With this, do you think we could/should drop the maxTokensToSend check above?

Reply:
I'm also curious if you think we should block deposits with amount == 0?

Reply:
It's a good question..... I guess is there any marginal value in blocking 0 if you can still send 1 wei deposits that might round to 0?